stdlib/Statistics/docs/src/index.md

```diff
@@ -4,22 +4,81 @@
 DocTestSetup = :(using Statistics)
 ```
 
-The Statistics module contains basic statistics functionality.
+The Statistics module contains basic statistics functionality: mean, median, quantiles,
+standard deviation, variance, skewness, kurtosis, correlation and covariance.
+Statistics can be weighted, and several weights types are distinguished to apply appropriate
```

Comment:
Perhaps this should read "several weight types" instead of "several weights types". If you're referring to the actual type then " several `AbstractWeights` types" maybe?

Reply:
Why "weight type"? The plural sounds more appropriate since weights only make sense as a series of values. Though "types of weights" might be better.

Reply:
It just sounded weird to me. I think "types of weights" is what sounds best.